nncf/experimental/netron.py

```diff
@@ -0,0 +1,197 @@
+        self.inputs = inputs
+        self.outputs = outputs
+
+    def as_xml_element(self) -> ET.Element:
+        node = ET.Element(Tags.NODE, id=self.node_id, name=self.name, type=self.type)
+        ET.SubElement(node, Tags.DATA, self.attrs)
+
+        if self.inputs:
+            input = ET.SubElement(node, Tags.INPUT)
+            for port in self.inputs:
+                input.append(port.as_xml_element())
+
+        if self.outputs:
+            output = ET.SubElement(node, Tags.OUTPUT)
+            for port in self.outputs:
+                output.append(port.as_xml_element())
+
+        return node
+
+
+class EdgeDesc:
+
+    def __init__(self,
+                 from_node: str,
+                 from_port: str,
+                 to_node: str,
+                 to_port: str):
+        self.from_node = from_node
+        self.from_port = from_port
+        self.to_node = to_node
+        self.to_port = to_port
+
+    def as_xml_element(self) -> ET.Element:
+        attrs = {
+            'from-layer': self.from_node,
+            'from-port': self.from_port,
+            'to-layer': self.to_node,
+            'to-port': self.to_port,
+        }
+        edge = ET.Element(Tags.EDGE, attrs)
+        return edge
+
+
+# TODO(andrey-churkin): Add support for `PortDesc.precision` param.
+def get_graph_desc(graph: NNCFGraph,
+                   include_fq_params: bool = False) -> Tuple[List[NodeDesc], List[EdgeDesc]]:
+    include_node = {}  # Dict[int, bool]
+    edges = []
+    for edge in graph.get_all_edges():
+        if not include_fq_params and edge.to_node.node_type == 'FakeQuantize' and edge.input_port_id != 0:
+            include_node[edge.from_node.node_id] = False
+            continue
+
+        edges.append(
+            EdgeDesc(
+                from_node=str(edge.from_node.node_id),
+                from_port=str(edge.output_port_id),
+                to_node=str(edge.to_node.node_id),
+                to_port=str(edge.input_port_id)
+            )
+        )
+
+    nodes = []
+    for node in graph.get_all_nodes():
+        if not include_node.get(node.node_id, True):
+            continue
+
+        inputs = []
+        for edge in graph.get_input_edges(node):
+            if not include_fq_params and node.node_type == 'FakeQuantize' and edge.input_port_id != 0:
+                continue
+
+            inputs.append(
+                PortDesc(
+                    port_id=str(edge.input_port_id),
+                    shape=edge.tensor_shape
+                )
+            )
+
+        outputs = []
+        for edge in graph.get_output_edges(node):
+            outputs.append(
+                PortDesc(
+                    port_id=str(edge.output_port_id),
+                    shape=edge.tensor_shape,
+                )
+            )
+
+        nodes.append(
+            NodeDesc(
+                node_id=str(node.node_id),
+                name=node.node_name,
+                type=node.node_type,
```

Comment:
```suggestion
                type=node.node_type.title(),
```
![image](https://user-images.githubusercontent.com/4014476/212350815-397415cd-208e-4dd9-a119-6ea02511dbed.png)
There is more chances to have pretty graph with type in a title case.
Probably, some mapping from NNCF types to Netron ones would be helpful in the future.